src/sentry/migrations/0067_remove_tracked_superusers.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-20 20:03
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def remove_tracked_superuser_views(apps, schema_editor):
+    """
+    We recently added code to only track alert views of people in the org + member of associated alert projects.
+    This migration removes all of the views we've tracked before adding this change (i.e. superuser views of orgs we're not a part of).
+    """
+    IncidentSeen = apps.get_model("sentry", "IncidentSeen")
+    IncidentProject = apps.get_model("sentry", "IncidentProject")
+    tracked_views = IncidentSeen.objects.all()
+    for tracked_view in tracked_views:
+        tracked_user = tracked_view.user
+        tracked_incident = tracked_view.incident
```

Comment:
These will emit additional queries. You might want to eager load these associations.